### PR TITLE
COMMERCE-8300 | master

### PR DESCRIPTION
**Pull request by bbqbronson**
[COMMERCE-8300](https://issues.liferay.com/browse/COMMERCE-8300)
Tests: CPCommerceChannels#viewOrderPreviewWithJasperTemplate, CPCommerceChannels#viewAndDownloadCustomJasperPrintTemplate, CPCommerceChannels#uploadCustonJasperPrintTemplate

### CODE REVIEW

**Comment by nikki-pru** [comment]
@bronsonsoda please check the committed changes. There are conflicting files. Let me know if you need help cleaning this up.

**Comment by bronsonsoda** [comment]
ci:test:sf

**Comment by bronsonsoda** [comment]
ci:test:sf

**Comment by nikki-pru** [comment]
ci:test:relevant

**Comment by nikki-pru** [comment]
ci:test:stable

**Comment by nikki-pru** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrders.path [hunk 1]
```diff
@@ -215,6 +215,31 @@
...
+	<td></td>
+</tr>
+<tr>
+	<td>PRINT_ORDER_TEMPLATE_FILE_NAME</td>
+	<td>//span[@id='_com_liferay_commerce_channel_web_internal_portlet_CommerceChannelsPortlet_fileEntryNameInput' and contains(.,'${value1}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PLACED_ORDERS_ELLIPSIS</td>
+	<td>//a[@class='component-action direction-right dropdown-toggle btn btn-lg btn-monospaced btn-primary position-fixed thumb-menu']</td>
```
Comment (nikki-pru) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrders.path:235]: This looks prone to failure due to the long @class.
check [icon.path](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Icon.path) for something reusable.

**Review by nikki-pru — COMMENTED** [review]
Great job on using macros! please see changes above.

**Comment by bronsonsoda** [comment]
@nikki-pru should be good now 🤔 I rebased, not sure if it is taking a while to update

**Comment by bronsonsoda** [comment]
it was because I forgot to push 😅

**Comment by nikki-pru** [comment]
ci:test:commerce-functional

**Comment by nikki-pru** [comment]
ci:test:relevant

**Comment by nikki-pru** [comment]
Testing on https://github.com/nikki-pru/liferay-portal/pull/350

**Pull request closed without merging**